packages/xo-web/src/xo-app/xoa/licenses/proxies.js

```diff
@@ -53,7 +53,7 @@ class ProxyLicensesForm extends Component {
         <ActionButton
           btnStyle='primary'
           className='ml-1'
-          disabled={licenseId === 'none'}
+          disabled={licenseId === 'none' || item.vmUuid === undefined}
```

Comment:
Or maybe add a tooltip if the button is disabled ?

Reply:
That can work too but I think that in that case, it doesn't even make sense to show the license selector if there's no way to bind a license to that proxy anyway. And since it's not a normal case (a proxy should have a VM UUID, right?), then an error message should be shown.

Reply:
It's not an error if the proxy doesn't have a VM UUID because it can have the VM IP. 
- if the proxy has no VM UUID : display a message to contact us to link a license
- if there is no license : disable a button
What do you think?